internal_error: ca_assert(is_list(value)) failed in list.cpp line 339
